src/bin/tikv-server.rs

```diff
@@ -97,9 +97,12 @@ fn build_raftkv(matches: &Matches,
     opts.set_max_write_buffer_number(5);
     opts.set_min_write_buffer_number_to_merge(2);
     opts.set_max_background_compactions(3);
-    opts.set_max_bytes_for_level_base(512 * 1024 * 1024);
+    opts.set_max_bytes_for_level_base(64 * 1024 * 1024);
```

Comment:
The size of level 0 is about 10MB and the size of level 1 should be similar to the size of level 0. So max_bytes_for_level_base should be Size_of_Level0_file * level0_file_num_compaction_trigger. The default value of level0_file_num_compaction_trigger is 4. So I set max_bytes_for_level_base to 60MB.  512Mb is too big.

Refer:
1. Siying Dong's suggestion in email. 
2. https://github.com/facebook/rocksdb/wiki/RocksDB-Tuning-Guide
"We usually reommend making L0->L1 as fast as possible by making the size of level 0 similar to size of level 1."